lib/shopify_api/resources/fulfillment_request.rb

```diff
@@ -0,0 +1,11 @@
+module ShopifyAPI
+  class FulfillmentRequest < Base
+    init_prefix :order
+
+    def order_id
+      @prefix_options[:order_id]
+    end
+
+    def cancel; load_attributes_from_response(post(:cancel, {}, only_id)); end
```

Comment:
Gotchya, so I should add a method to cancel the associated fulfillment as well?

Since the object we're operating on is a fulfillment request isn't it clear enough that ````fulfillment_request.cancel```` would cancel it (ie the fulfillment request)? And then we could add a ````fulfillment_request.cancel_fulfillment```` method